crontasks/list.go

```diff
@@ -1,21 +1,29 @@
 package crontasks
 
 import (
+	"os"
+
 	"github.com/Scalingo/cli/config"
+	httpclient "github.com/Scalingo/go-scalingo/v4/http"
+	"github.com/Scalingo/go-utils/errors"
 	"github.com/olekukonko/tablewriter"
-	"github.com/pkg/errors"
-	"os"
+	"gopkg.in/errgo.v1"
 )
 
 func List(app string) error {
 	client, err := config.ScalingoClient()
 	if err != nil {
-		return errors.Wrap(err, "fail to get Scalingo client")
+		return errgo.Notef(err, "fail to get Scalingo client")
 	}
 
 	cronTasks, err := client.CronTasksGet(app)
 	if err != nil {
-		return errors.Wrap(err, "fail to get cron tasks")
+		rootError := errors.ErrgoRoot(err)
+		if !httpclient.IsRequestFailedError(rootError) || rootError.(*httpclient.RequestFailedError).Code != 404 {
```

Comment:
What happen if the `rootError` is not a `httpclient.RequestFailedError` ?
Does it throw an error because `.Code` cannot be accessible ?

Reply:
If `rootError` is not a `httpclient.RequestFailedError`, the condition evaluates to `true`. This is a _or_ condition, not a _and_.

Reply:
Oh, I read too quickly.. mb